src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,76 @@
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  height: 60px;
+
+  &__logo {
+    display: inherit;
+    align-self: center;
+  }
+
+  &__menu {
+    display: inherit;
+  }
+
+  &__list {
+    display: inherit;
+  }
+
+  &__item {
+    display: inherit;
+    position: relative;
+    margin-left: 20px;
+    font-weight: 500;
+    text-transform: uppercase;
+    font-size: 12px;
+    line-height: 14px;
+    &:first-child {
+      margin-left: 0;
+    }
+
+  }
+
+  &__link {
+    font-family: "Roboto", sans-serif;
+    display: inherit;
+    align-items: center;
+    height: 60px;
```

Comment:
why do you need to write height: 60px twice?

Reply:
![image](https://user-images.githubusercontent.com/105974422/180201924-cf638773-0583-4f25-87b3-394690232f6c.png)
blue line position depends on this height

Reply:
you can add paddings to fix it